client/tests/webdriver/specs/createNewPerson.spec.js

```diff
@@ -9,6 +9,8 @@ describe('Create new Person form page', () => {
 
     beforeEach('On the create a new user page...', () => {
         CreatePerson.open()
+        CreatePerson.form.waitForExist()
+        CreatePerson.form.waitForVisible()
```

Comment:
Shouldn't this also be added to the editPosition specs?

Reply:
Will do.